<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>401</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>243</width>
    <height>401</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>243</width>
    <height>401</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>SHaRK v0.16.7.10</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pic/256x256_WhiteLogo.png</normaloff>pic/256x256_WhiteLogo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-2</y>
      <width>241</width>
      <height>301</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Primary processing</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="0">
       <widget class="QLineEdit" name="un_tags_Line">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the &lt;span style=&quot; font-weight:600;&quot;&gt;undesired&lt;/span&gt; sequence &lt;span style=&quot; font-weight:600;&quot;&gt;tags&lt;/span&gt; here&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="placeholderText">
         <string>Undesired tag(s)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="inputfilepathLine">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the &lt;span style=&quot; font-weight:600;&quot;&gt;input&lt;/span&gt; file path(s)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Open file(s)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="deleterepeatsBox">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Delete repeateated &lt;/span&gt;sequences according to &lt;span style=&quot; font-weight:600;&quot;&gt;taxon name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Delete repeats</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="referencepathLine">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the &lt;span style=&quot; font-weight:600;&quot;&gt;reference&lt;/span&gt; sequence file path&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Reference sequence</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QToolButton" name="referencepathButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose the &lt;span style=&quot; font-weight:600;&quot;&gt;reference&lt;/span&gt; sequence file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="req_tags_Line">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the &lt;span style=&quot; font-weight:600;&quot;&gt;required&lt;/span&gt; sequence &lt;span style=&quot; font-weight:600;&quot;&gt;tags&lt;/span&gt; here&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Required tag(s)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="inputfilepathButton">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose the &lt;span style=&quot; font-weight:600;&quot;&gt;input&lt;/span&gt; file(s)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="outputfilepathLine">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set &lt;span style=&quot; font-weight:600;&quot;&gt;output&lt;/span&gt; directory&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Output directory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="outputfilepathButton">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose &lt;span style=&quot; font-weight:600;&quot;&gt;output&lt;/span&gt; directory&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="uniteBox">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create &lt;span style=&quot; font-weight:600;&quot;&gt;united sets&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Unite files</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Alignment</string>
     </attribute>
     <widget class="QSpinBox" name="gap_spinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set &lt;span style=&quot; font-weight:600;&quot;&gt;gap open penalty&lt;/span&gt; for &lt;span style=&quot; font-weight:600;&quot;&gt;MUSCLE alignment&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="value">
       <number>-200</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="iters_spinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set &lt;span style=&quot; font-weight:600;&quot;&gt;maximum&lt;/span&gt; number of &lt;span style=&quot; font-weight:600;&quot;&gt;iterations&lt;/span&gt; for &lt;span style=&quot; font-weight:600;&quot;&gt;MUSCLE alignment&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="diagonal_checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use &lt;span style=&quot; font-weight:600;&quot;&gt;diagonal optimization&lt;/span&gt; during &lt;span style=&quot; font-weight:600;&quot;&gt;MUSCLE alignment&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Diagonal optimization</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="silent_checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose, whether to show the &lt;span style=&quot; font-weight:600;&quot;&gt;command line output&lt;/span&gt; of &lt;span style=&quot; font-weight:600;&quot;&gt;MUSCLE algorithm&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Silent performance</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Gap open penalty</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Maximum number of iterations</string>
      </property>
     </widget>
     <widget class="QComboBox" name="range_comboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the best &lt;span style=&quot; font-weight:600;&quot;&gt;truncation range&lt;/span&gt; calculation &lt;span style=&quot; font-weight:600;&quot;&gt;method&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>minimum</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mean</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>median</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>maximum</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>percentile</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="trunc_checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Calculate the &lt;span style=&quot; font-weight:600;&quot;&gt;best truncation range&lt;/span&gt; for the given data sets&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Truncation assistance</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Range calculation</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="perc_doubleSpinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the &lt;span style=&quot; font-weight:600;&quot;&gt;percentile&lt;/span&gt; for best &lt;span style=&quot; font-weight:600;&quot;&gt;truncation range&lt;/span&gt; calculation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="suffix">
       <string>%</string>
      </property>
      <property name="maximum">
       <double>50.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
      <property name="value">
       <double>25.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Range percentile</string>
      </property>
     </widget>
     <widget class="QComboBox" name="align_opt_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>Full set</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tax subsets</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Even subsets</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Alignment option</string>
      </property>
     </widget>
     <zorder>range_comboBox</zorder>
     <zorder>gap_spinBox</zorder>
     <zorder>iters_spinBox</zorder>
     <zorder>diagonal_checkBox</zorder>
     <zorder>silent_checkBox</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>trunc_checkBox</zorder>
     <zorder>label_4</zorder>
     <zorder>perc_doubleSpinBox</zorder>
     <zorder>label_5</zorder>
     <zorder>align_opt_comboBox</zorder>
     <zorder>label_6</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Resampling</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="checksourceBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save &lt;span style=&quot; font-weight:600;&quot;&gt;source version&lt;/span&gt; of &lt;span style=&quot; font-weight:600;&quot;&gt;processed file&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>Save source file</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QCheckBox" name="showresultsBox">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show &lt;span style=&quot; font-weight:600;&quot;&gt;performance report&lt;/span&gt; with program &lt;span style=&quot; font-weight:600;&quot;&gt;run stats&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Performance report</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QSpinBox" name="copiesnumberSpinBox">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set number of file &lt;span style=&quot; font-weight:600;&quot;&gt;copies&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Copies number</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QComboBox" name="deletionoptionCombo">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose the &lt;span style=&quot; font-weight:600;&quot;&gt;way of definig&lt;/span&gt; the &lt;span style=&quot; font-weight:600;&quot;&gt;replica composition&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <item>
         <property name="text">
          <string>Delete sequences</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Set sample size</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QSpinBox" name="deletionnumberSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose the sequences &lt;span style=&quot; font-weight:600;&quot;&gt;deletion/subsampling&lt;/span&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QDoubleSpinBox" name="deletionpercentDoubleSpinBox">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose the sequences &lt;span style=&quot; font-weight:600;&quot;&gt;deletion/subsampling&lt;/span&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;percent&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QRadioButton" name="percentageRadio">
        <property name="text">
         <string>Percentage</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QRadioButton" name="numberRadio">
        <property name="text">
         <string>Number</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="split_checkBox">
        <property name="text">
         <string>Split by taxon</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QDialogButtonBox" name="goDialogButtonBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Run&lt;/span&gt; the programm or&lt;span style=&quot; font-weight:600;&quot;&gt; reset the values&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedKingdom"/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
    </property>
    <property name="centerButtons">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The &lt;span style=&quot; font-weight:600;&quot;&gt;progress&lt;/span&gt; of &lt;span style=&quot; font-weight:600;&quot;&gt;file processing&lt;/span&gt; is displayed here&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>numberRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>deletionnumberSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>percentageRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>deletionpercentDoubleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trunc_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>range_comboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
